src/provider/view/components/NotificationGroup/NotificationGroup.tsx

```diff
@@ -27,11 +27,11 @@ export function NotificationGroup(props: NotificationGroupProps) {
     }
 
     if (props.notifications.length > 0) {
-        const sortedNotifications = props.notifications.sort((a: INotification, b: INotification) => {
-            return a.date > b.date ? -1 : a.date < b.date ? 1 : 0;
+        const sortedNotifications = props.notifications.sort((a: StoredNotification, b: StoredNotification) => {
+            return a.notification.date > b.notification.date ? -1 : a.notification.date < b.notification.date ? 1 : 0;
```

Comment:
I know not your code, but do we expect this to work? We don't need to do valueOf() here?

Reply:
It's an interesting question. The engine does seem to handle date subtraction just fine, but typescript doesn't seem to like it.

As for whether it works, I've tested sorting manually and it seems to work ok, but hard to say with 100% confidence. As above, this is likely to change soon when the multi-window redux stuff comes in, so I'd say not worth putting too much effort in for this story.

Reply:
Date objects do support inequality operators, so this is safe.

Though `return b.notification.date.getTime() - a.notification.date.getTime();` would be a much simpler way of doing this - but as Tom points out will likely all get re-worked fairly soon anyway.

Reply:
In a different spot I think we use valueOf just to make typescript happy, but getTime would work equally well